lib/search.rb

```diff
@@ -791,6 +791,8 @@ def posts_query(limit, opts = nil)
     posts = Post.where(post_type: Topic.visible_post_types(@guardian.user))
       .joins(:post_search_data, :topic)
       .joins("LEFT JOIN categories ON categories.id = topics.category_id")
+      .joins("LEFT JOIN topic_tags ON topic_tags.topic_id = topics.id")
```

Comment:
hmmm why are we joining these tables if @search_context is not a Tag?

Reply:
Good point, will do this conditionally along with the category join

Reply:
Actually, never mind, the categories join is used for a few other things, so I'll only conditionally join the tag tables